### ISSUE TITLE
Looking for a way to search/find events containing several tags (/events/index/searchtag)

### ISSUE THREAD

**Issue by obert01** [MISP/MISP#2416]
Hello,

In the web interface, if I click to modify the search filters, I can easily search events matching one or more tags. 

This generates URL like:
https://mymisp.org/events/index/searchtag:circl:incident-classification="malware"|circl:incident-classification="vulnerability"

But I have the impression an option to match events containing all of the given tags is missing. Let's say: circl:incident-classification="malware" and (&) circl:incident-classification="vulnerability". 

Am I missing something? Does this function exist in the web UI? Is it possible to include it in a future version?

Many thanks

P.S.: I know this function is available in certain of the automation exports (/events/xml/download, /events/csv/download, ...), but I would really need to have it in the web UI.

**Comment by iglocska** [comment]
It doesn't exist but good idea!

**Comment by vmapps** [comment]
Hello,

API documentation says that "You can also chain several tag commands together with the '&&' operator." but have not tested it yet.
However, "AND" boolean operator does not seem to be supported in the web GUI, not the PyMISP library.

Please also note that when using API "be aware the colons (:) cannot be used in the tag search. Use semicolons instead (the search will automatically search for colons instead)."

See also #2726

**Comment by fossilos** [comment]
YES, PLEASE PLEASE PLEASE add this feature to the GUI. It's crazy not being able to use the AND operator when searching for events with multiple tags. Please prioritize!!!

**Comment by github-germ** [comment]
https://github.com/MISP/PyMISP/issues/316
